<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculadora</class>
 <widget class="QMainWindow" name="Calculadora">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h1&gt;Calculadora de Logaritmo&lt;/h1&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calcular">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
background-color: #090;
color: #fff;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>195</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="base_logaritmando">
       <property name="styleSheet">
        <string notr="true">font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Base e Logaritmando</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="base_logaritmo">
       <property name="styleSheet">
        <string notr="true">font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Base e Logaritmo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="logaritmo_logaritmando">
       <property name="styleSheet">
        <string notr="true">font-size: 15px;
</string>
       </property>
       <property name="text">
        <string>Logaritmo e Logaritmando</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="txt_logaritmando">
       <property name="styleSheet">
        <string notr="true">font-size: 16px;</string>
       </property>
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txt_base">
       <property name="styleSheet">
        <string notr="true">font-size: 16px;</string>
       </property>
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="input_num1">
       <property name="styleSheet">
        <string notr="true">font-size: 12px;</string>
       </property>
       <property name="text">
        <string>Insira</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="input_num2">
       <property name="styleSheet">
        <string notr="true">font-size: 12px;</string>
       </property>
       <property name="text">
        <string>Insira</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size: 16px;</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txt_logaritmo">
       <property name="styleSheet">
        <string notr="true">font-size: 16px;</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>191</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="num1">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="num2">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
